packages/frontend/src/components/ProjectLink.tsx

```diff
@@ -19,33 +19,33 @@ export function ProjectLink({ project, type }: Props) {
       : `/bridges/projects/${project.slug}`
   return (
     <>
-      <a className="ProjectLink" href={href}>
+      <a className="ProjectCell--link" href={href}>
```

Comment:
If you are not planning on rewriting the styles of `ProjectLink` in tailwind I would just leave them untouched.